source/_patterns/01-atoms/dropdown/_dropdown.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../00-base/config';
+
+// To change default bootstrap nav variables,
+// copy from ~bootstrap/scss/variables to here
+
+$dropdown-link-color: $gray-900;
+@import "~bootstrap/scss/dropdown";
+@import "~bootstrap/scss/button-group";
```

Comment:
Need to make another component for button-group and have it import in index.js to avoid potential repeats